docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+name: inno-microservices
+
+services:
+  postgres:
+    image: postgres:latest
+    container_name: postgres
+    environment:
+      POSTGRES_DB: inno-db
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+      - 5433:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres -d inno-db"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    volumes:
+      - .\volumes\postgres\data:/var/lib/postgresql/data
```

Comment:
Using ./volumes instead of .\volumes is the correct cross-platform approach and will ensure the Docker Compose file works consistently across all operating systems.

Check out other similar places